sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -1130,6 +1130,10 @@ public DigitalTwinsServiceVersion getServiceVersion() {
      *
      * {@codesnippet com.azure.digitaltwins.core.asyncClient.query#String#String}
      *
+     * Note that there may be a delay between before changes in your instance are reflected in queries.
```

Comment:
The note seemed a little hard to understand. This might be better - 'there may be a delay between changes in your instance reflecting in queries'?

Reply:
I used the same language that is used in the official document that is being linked

Reply:
Going by the previous PR discussion, if we remove 'between' from the wording, it should be good. "Note that there may be a delay before changes in your instance are reflected in queries."